src/tests/db/controller/article.test.js

```diff
@@ -0,0 +1,66 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../../server';
+
+// configure chai to use expect
+chai.use(chaiHttp);
+const { expect } = chai;
+
+const userRequestObject = {
+  username: 'boxunivers',
+  email: 'lilly@gmail.com',
+  password: 'hhrtuyhgty5t678',
+};
+const articleRequestObject = {
+  title: 'the hope of life',
+  description: 'in times of trouble, where do we find hope',
+  body: 'when our peace is no more, when friends are no where to be found. we need to do back to our source.',
+  images: 'image1.jpg, image2.png',
+  tags: 'hope, life, source.'
+};
+
+let xAccessToken = '';
+
+describe('CREATE ARTICLE', () => {
+  it('should create user, to enable us use jwt token', (done) => {
+    chai.request(app)
+      .post('/api/v1/users')
+      .send(userRequestObject)
+      .end((err, res) => {
+        expect(res.status).to.equal(201);
```

Comment:
let's test for the response data.

Reply:
thanks, I have tested for the response data